include/boost/gil/channel.hpp

```diff
@@ -596,64 +604,6 @@ void swap(const boost::gil::packed_dynamic_channel_reference<BF,NB,M> x, const b
 }
 }   // namespace std
 
-namespace boost { namespace gil {
-///////////////////////////////////////////
-////
-////  Built-in channel models
-////
-///////////////////////////////////////////
-
-/// \defgroup bits8 bits8
```

Comment:
Can we keep these comments? Rewritten for std types?

Reply:
Ah, I see you moved them to typedefs.hpp

Reply:
Yup, the commit log says
> (...) move from channel.hpp to typedefs.hpp for easier access